composer.json

```diff
@@ -21,18 +21,18 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.4",
         "laminas/laminas-console": "^2.6",
         "laminas/laminas-dom": "^2.6",
         "laminas/laminas-eventmanager": "^2.6.2 || ^3.0",
         "laminas/laminas-http": "^2.8.3",
         "laminas/laminas-mvc": "^3.0",
         "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
-        "laminas/laminas-stdlib": "^2.7 || ^3.0",
+        "laminas/laminas-stdlib": "^3.3",
```

Comment:
Revert this please; see my comment about the PHP constraint.

Reply:
done